src/cookbooks/bigquery/access.md

```diff
@@ -101,22 +113,30 @@ pip install google-cloud-bigquery
 python -c 'from google.cloud import bigquery; print([d.dataset_id for d in bigquery.Client().list_datasets()])'
 ```
 
-## From Spark
+### Spark
+
+> This method requires [BigQuery Access](#BigQuery-Access-Request) to be provisioned.
+
 We recommend the [Storage API Connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector) for accessing
 BigQuery tables in Spark as it is the most modern and actively developed connector. It works well with the BigQuery
 client library which is useful if you need to run arbitrary SQL queries (see example Databricks notebook) and load their
 results into Spark.
 
-### On Databricks
-The `shared_serverless_python3` cluster is configured with shared default GCP credentials that are automatically picked
-up by BigQuery client libraries. It also has the Storage API Connector library added as seen in the example
-[Python notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
+### Databricks
```

Comment:
Preserve relative heading level:
```suggestion
#### Databricks
```

Reply:
This was a deliberate change, as some of the remarks about Spark (e.g. special access provisioning) do not apply to databricks. I'm going to keep this one as-is, if you have objections we can take a look in a follow-up.

Reply:
Actually, on second look, you were right. I think it was just the section above that needed adjusting. Fixed now (I think)